Guides/TLS.md

```diff
@@ -0,0 +1,78 @@
+# TLS/SSL and MongoSwift
+
+MongoSwift supports connecting to MongoDB over TLS/SSL. This guide covers the installation requirements and configuration options for enabling this functionality in MongoSwift. See the [server documentation](https://docs.mongodb.com/manual/tutorial/configure-ssl/) to configure MongoDB to use TLS/SSL.
```

Comment:
I think we have to be a little careful about using the name `MongoSwift` now that we have two modules and the library isn't named `MongoSwift`. It might lead you to think the sync API doesn't support TLS. I think it's probably ok to just say "the driver".